### DISCUSSION TITLE
help? haugene/transmission-openvpn K3s rpi

### DISCUSSION THREAD

**Discussion by i-rv** [haugene/docker-transmission-openvpn#2034] [Q&A] [answered]
Getting one or the other error

Oct 23 13:29:37 kube-master k3s[877]: E1023 13:29:37.663333     877 pod_workers.go:190] "Error syncing pod, skipping" err="failed to \"StartContainer\" for \"transmission-openvpn\

Or

pi@kube-master:~ $ kubectl get pods -o wide -n media
NAME                                                READY   STATUS         RESTARTS   AGE   IP           NODE          NOMINATED NODE   READINESS GATES
transmission-transmission-openvpn-75f849c8b-kwcqp   0/1     ErrImagePull

image:
  repository: "haugene/transmission-openvpn"
  tag: "latest-armhf"
  
  
  
  Omitted "-armhf" and still get the 1st error. Using TorGuard vpn in config file, if that's any help.
  
  Last but not the least: I am just 2 days new into this world!

**Comment by pkishino** [comment, marked as answer]
there are no -armhf specific builds anymore, you should just be pulling :latest
regarding the first one I am not sure, possibly config error or such. I don't use K8s

**Reply by pkishino** [reply]
I looked again, your setting is wrong..
repository should be dockerhub, image should be haugene/transmission-openvpn
